.github/workflows/ci-build.yml

```diff
@@ -28,21 +28,32 @@ jobs:
             ${{ runner.os }}-maven-
       - name: "Prepare container reuse"
         run: |
-          echo 'testcontainers.reuse.enable=true' > "$HOME/.testcontainers.properties"
+          echo 'testcontainers.reuse.enable=true' > "/home/runner/.testcontainers.properties"
       - name: "Run tests and build with Maven"
         run: |
           echo
-          echo "-------------------------------------------------------------"
-          echo "Running matrix test with dockerdb_version $DOCKER_DB_VERSION"
-          echo "-------------------------------------------------------------"
+          echo "--------------------------------------------------------------"
+          echo " Running matrix test with dockerdb_version $DOCKER_DB_VERSION "
+          echo "--------------------------------------------------------------"
           echo
+          echo "Support information archives are written to $SUPPORT_DIR"
+          echo
+          mkdir -p "$SUPPORT_DIR"
           mvn --batch-mode --update-snapshots clean verify sonar:sonar --file pom.xml \
           -DtrimStackTrace=false -Dsonar.organization=exasol -Dsonar.host.url=https://sonarcloud.io \
           -Dsonar.login="$SONAR_TOKEN" \
           -Dcom.exasol.dockerdb.image="$DOCKER_DB_VERSION" \
+          -Dcom.exasol.containers.support_information_target_dir="$SUPPORT_DIR" \
+          -Dcom.exasol.containers.monitored_exit=EXIT_ANY \
           -Pslow
         env:
           DOCKER_DB_VERSION: ${{ matrix.docker_db_version }}
           GITHUB_OAUTH: ${{ secrets.GITHUB_TOKEN }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
\ No newline at end of file
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+          SUPPORT_DIR: ${{ github.workspace }}/target/support
```

Comment:
I think  you can directly use this expression in the above command. No need for an env var

Reply:
Possibly. But I find it cleaner to keep GitHub artifacts out of the shell script part if possible.